clickRegion.lua

```diff
@@ -0,0 +1,42 @@
+local PrimeUI = require "util" -- DO NOT COPY THIS LINE
+local expect = require "cc.expect".expect -- DO NOT COPY THIS LINE
+-- Start copying below this line. --
+
+--- Creates a clickable region on screen without any content.
+---@param win window The window to draw on
+---@param x number The X position of the button
+---@param y number The Y position of the button
+---@param width number The width of the inner box
+---@param height number The height of the inner box
+---@param action function|string A function to call when clicked, or a string to send with a `run` event
+---@param periphName string|nil The name of the monitor peripheral, or nil (set if you're using a monitor - events will be filtered to that monitor)
+function PrimeUI.clickRegion(win, x, y, width, height, action, periphName)
+	expect(1, win, "table")
+	expect(2, x, "number")
+	expect(3, y, "number")
+	expect(4, width, "number")
+	expect(5, height, "number")
+	expect(6, action, "function", "string")
+	periphName = expect(7, periphName, "string", "nil")
+	-- Get the screen position and add a click handler.
+	local screenX, screenY = PrimeUI.getWindowPos(win, x, y)
+	PrimeUI.addTask(function()
+		local buttonDown = false
+		while true do
+			local event, button, clickX, clickY = os.pullEvent()
+            if event == "monitor_touch" and periphName == button
+				or event == "mouse_up" and button == 1 and periphName == nil then
```

Comment:
As stated above, please fix indentation, and use parentheses around `or` clauses:
```suggestion
            if (event == "monitor_touch" and periphName == button)
                or (event == "mouse_down" and button == 1 and periphName == nil) then
```